### PR TITLE
Update styles for mod/exotic pickers

### PR DESCRIPTION
**Pull request by bhollis**
Cleaned up the styles for these to make better use of the space, then consolidated on common utility components to remove duplication. I decided that perk icons weren't that useful for the exotic picker and that they distracted from reading. I also moved general (stat) mods below bucket-specific mods.

Before:
Screenshot 2023-06-01 at 11 47 50 AM
After:
Screenshot 2023-06-01 at 11 47 41 AM

Before:
Screenshot 2023-06-01 at 11 47 59 AM
After:
Screenshot 2023-06-01 at 11 47 33 AM

Before:
Screenshot 2023-06-01 at 11 49 04 AM
After:
Screenshot 2023-06-01 at 11 48 46 AM

### CODE REVIEW

**Review by vivekhnz — COMMENTED** [review]
Nice cleanup! I'm generally on board, although I think the different text 'tiers' in the tiles could use a bit more distinction from eachother.

I'm finding it a bit difficult to quickly differentiate between the mod description and the mod name, or the exotic perk description and the exotic perk name. I don't think changing **one of** the font weight/size/colour is enough - changing two of those facets seems to work better IMO.

Some illustrations and possible ideas:
dim-plug-picker-styles

File: src/app/dim-ui/TileGrid.m.scss [hunk 1]
```diff
@@ -37,6 +37,7 @@
   display: flex;
   flex-direction: column;
   gap: 4px;
+  //color: var(--theme-text-secondary);
```
Comment (vivekhnz) [inline src/app/dim-ui/TileGrid.m.scss:40]: Intentional?

**Review by vivekhnz — APPROVED** [review]
Looks great!

File: src/app/dim-ui/TileGrid.m.scss (hunk 1 above)
Comment (bhollis) [reply src/app/dim-ui/TileGrid.m.scss:40]: Haha nope!

**Pull request merged**